float8_experimental/float8_dynamic_utils.py

```diff
@@ -53,64 +53,6 @@ def backward(ctx, gradY):
         return fp8_tensor, None
```

Comment:
do you think there is much value in putting this into dynamic utils vs into utils?

Reply:
I think the git history is a bit cleaner this way which is why it made sense to me for this PR (minimizes changes to utils.py).  I'm not opposed to consolidate in a future PR, don't feel strongly one way or the other.